bolt-contracts/test/BoltChallenger.t.sol

```diff
@@ -51,7 +56,32 @@ contract BoltChallengerTest is Test {
         vm.pauseGasMetering();
         (target, targetPK) = makeAddrAndKey("target");
 
+        uint48 epochDuration = 1 days;
+        uint48 slashingWindow = 7 days;
+        uint48 maxChallengeDuration = 7 days;
+        bool allowUnsafeRegistration = true;
+        uint256 challengeBond = 1 ether;
+        uint256 blockhashEvmLookback = 256;
+        uint256 justificationDelay = 32;
+        uint256 eth2GenesisTimestamp = 1_606_824_023;
+        uint256 slotTime = 12;
```

Comment:
These are copy-pasted many times, any chance we can source it from a JSON file? I think Nico has done it in the past

Reply:
Done in latest commit! Required a bit of scaffolding but lmk what you think